reference/configuration/framework.rst

```diff
@@ -1152,7 +1152,10 @@ utf8
 
 **type**: ``boolean`` **default**: ``false``
 
-When this option is set to ``true``, route patterns can include UTF-8 characters.
+When this option is set to ``true``, the regular expressions used in the
+:ref:`requirements of route parameters <routing-requirements>` will match any
+UTF-8 character when using ``.``, instead of matching only a single byte.
```

Comment:
The `u` modifier influences a bit more than just the `.`, doesn't it? IIRC it also changes how the `\w` class matches for example.